scripts/get

```diff
@@ -78,16 +78,31 @@ verifySupported() {
 # checkDesiredVersion checks if the desired version is available.
 checkDesiredVersion() {
   # Use the GitHub releases webpage for the project to find the desired version for this project.
-  local release_url="https://github.com/helm/helm/releases/${DESIRED_VERSION:-latest}"
+  local latest_release_url="https://github.com/helm/helm/releases/latest"
+  local release_url
+  if [ "x$DESIRED_VERSION" == "x" ]; then
+    if type "curl" > /dev/null; then
+      release_url=$(curl -Ls -o /dev/null -w %{url_effective} $latest_release_url)
+    elif type "wget" > /dev/null; then
+      release_url=$(wget $latest_release_url --server-response -O /dev/null 2>&1 | awk '/^  Location: /{DEST=$2} END{ print DEST}')
+    fi
+  else
+    release_url="https://github.com/helm/helm/releases/${DESIRED_VERSION}"
+  fi
+
+  local status_code
   if type "curl" > /dev/null; then
-    TAG=$(curl -SsL $release_url | awk '/\/tag\//' | grep -v no-underline | grep "<a href=\"/helm/helm/releases" | head -n 1 | cut -d '"' -f 2 | awk '{n=split($NF,a,"/");print a[n]}' | awk 'a !~ $0{print}; {a=$0}')
+    status_code=$(curl --write-out %{http_code} --silent --output /dev/null $release_url)
```

Comment:
This step can be removed. We'll be using the release URL to fetch the assets later in the script so there's no need to do an empty status code check here. Once we have the right release URL, let's just parse the information out.

Reply:
Ok. I will remove it. Thanks

Reply:
if DESIRED_VERSION set by user is invalid, we will have message like this
 ```
SHA sum of /var/folders/ts/yc5gs81d46n3mnk4nztl8sxc0000gn/T/helm-installer-XXXXXX.0MniFZZI/helm-2.3-darwin-amd64.tar.gz does not match. Aborting.
```
instead of 
```
Cannot determine 2.3 tag.
```
is it ok?